src/App/Fossa/Main.hs

```diff
@@ -107,6 +110,31 @@ experimentalLicenseScanCommand = command "experimental-license-scan" (info runIn
     runInit :: Parser (IO ())
     runInit = pure $ putStrLn "The 'experimental-license-scan' has been deprecated and renamed to 'license-scan'. Please use the 'license-scan' command instead."
 
+feedbackCommand :: Mod CommandFields (IO ())
+feedbackCommand = command "feedback" (info feedbackPrompt $ progDescDoc $ formatStringToDoc "Provide feedback on your fossa-cli experience, submit feature requests, and report bugs/issues")
+  where
+    feedbackPrompt :: Parser (IO ())
+    feedbackPrompt =
+      pure $
+        putDoc $
+          vsep
+            [ newlinePreceding "At FOSSA, we are committed to delivering an exceptional user experience and are continously working towards improving our product."
+            , "Your feedback is crucial in shaping our ongoing efforts to innovate and provide an even better user experience!"
+            , ""
+            , "* Report bugs and issues at:"
+            , newlineTrailing . newlinePreceding $ indent 4 $ pretty supportUrl
+            , "* Submit feature requests to:"
+            , newlineTrailing . newlinePreceding $ indent 4 "support@fossa.com"
+            , "* Provide feedback on overall cli experience at:"
+            , newlineTrailing . newlineTrailing . newlinePreceding $ indent 4 "https://docs.google.com/forms/d/e/1FAIpQLSdfz4KX_j6lpCF0zICQaUd-Rjn_Vj-IelCSmqOii__fqyYVmg/viewform"
```

Comment:
Where does this go?

Reply:
Thinking about this more, I think this url being like this is not a good look. We should short link at least - we have go.fossa.com short links in route53

Reply:
It goes to a google form that I created

Yea I like the idea of having a short link. Is it not a good look in terms of a security standpoint or is it due to readability?